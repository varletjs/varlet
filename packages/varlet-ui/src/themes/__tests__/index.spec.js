import { expect } from 'vitest'
import Themes from '..'

test('test dark theme', () => {
  expect(Themes.dark).toMatchSnapshot()
})
