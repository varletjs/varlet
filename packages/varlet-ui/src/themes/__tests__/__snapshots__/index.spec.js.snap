// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test dark theme 1`] = `
{
  "--action-sheet-action-item-color": "#fff",
  "--action-sheet-background": "#1e1e1e",
  "--action-sheet-title-color": "#aaa",
  "--app-bar-color": "#272727",
  "--avatar-background-color": "#303030",
  "--avatar-border": "2px solid #1e1e1e",
  "--badge-default-color": "#555",
  "--bottom-navigation-background-color": "#272727",
  "--bottom-navigation-border-color": "#444",
  "--bottom-navigation-item-active-background-color": "#272727",
  "--bottom-navigation-item-inactive-color": "#BFBFBF",
  "--breadcrumb-inactive-color": "#aaa",
  "--button-default-color": "#303030",
  "--button-default-text-color": "#fff",
  "--card-background": "#303030",
  "--card-description-color": "#aaaaaa",
  "--card-outline-color": "rgba(255, 255, 255, 0.2)",
  "--card-subtitle-color": "#aaaaaa",
  "--card-title-color": "#ffffff",
  "--cell-border-color": "#545454",
  "--cell-description-color": "#aaa",
  "--checkbox-text-color": "#fff",
  "--checkbox-unchecked-color": "#fff",
  "--chip-default-color": "#555",
  "--chip-default-text-color": "#fff",
  "--collapse-background": "#303030",
  "--collapse-border-top": "thin solid rgba(255, 255, 255, 0.12)",
  "--collapse-text-color": "#fff",
  "--color-body": "#1e1e1e",
  "--color-danger": "#ef5350",
  "--color-disabled": "#404040",
  "--color-info": "#10afef",
  "--color-primary": "#4a7afe",
  "--color-success": "#10c48f",
  "--color-text": "#fff",
  "--color-text-disabled": "#757575",
  "--color-warning": "#ff8800",
  "--countdown-text-color": "#fff",
  "--date-picker-body-background-color": "#303030",
  "--date-picker-main-color": "#fff",
  "--day-picker-head-item-color": "#aaaaaa",
  "--dialog-background": "#1e1e1e",
  "--dialog-message-color": "#bbb",
  "--dialog-title-color": "#fff",
  "--divider-color": "rgba(255, 255, 255, 0.2)",
  "--divider-text-color": "#aaa",
  "--field-decorator-blur-color": "rgb(255, 255, 255, .7)",
  "--field-decorator-text-color": "#fff",
  "--floating-panel-background": "#272727",
  "--link-default-color": "#fff",
  "--menu-background-color": "#272727",
  "--menu-option-text-color": "#fff",
  "--menu-select-menu-background-color": "#272727",
  "--option-text-color": "#fff",
  "--pagination-hover-bg-color": "rgba(255, 255, 255, .15)",
  "--pagination-item-background": "#303030",
  "--pagination-list-active-bg-color": "#25293a",
  "--pagination-list-active-color": "#4a7afe",
  "--pagination-list-bg-color": "#303030",
  "--pagination-text-color": "#fff",
  "--paper-background": "#303030",
  "--picker-background": "#1e1e1e",
  "--picker-cancel-button-text-color": "#aaa",
  "--picker-header-color": "#fff",
  "--picker-mask-background-image": "linear-gradient(180deg, hsla(0, 0%, 12%, 0.9), hsla(0, 0%, 12%, 0.4)), linear-gradient(0deg, hsla(0, 0%, 12%, 0.9), hsla(0, 0%, 12%, 0.4))",
  "--picker-option-text-color": "#fff",
  "--picker-picked-border": "1px solid rgba(255, 255, 255, 0.12)",
  "--picker-title-text-color": "#fff",
  "--popup-content-background-color": "#1e1e1e",
  "--progress-label-color": "#fff",
  "--pull-refresh-background": "#303030",
  "--radio-text-color": "#fff",
  "--radio-unchecked-color": "#fff",
  "--result-background": "#303030",
  "--result-description-color": "#aaaaaa",
  "--result-empty-border-color": "rgba(232,229,229,0.3)",
  "--result-empty-color": "#adadad",
  "--result-question-border-color": "rgba(151,194,216,0.3)",
  "--result-question-color": "#7f8e96",
  "--result-title-color": "#ffffff",
  "--select-scroller-background": "#272727",
  "--skeleton-animation-background": "linear-gradient(
        90deg,hsla(0,0%,100%,0),hsla(0,0%,100%,.05),hsla(0,0%,100%,0))
      ",
  "--skeleton-avatar-background-color": "hsla(0,0%,100%,.12)",
  "--skeleton-card-background-color": "hsla(0,0%,100%,.12)",
  "--skeleton-title-background-color": "hsla(0,0%,100%,.12)",
  "--step-content-active-color": "#fff",
  "--step-content-color": "rgba(255, 255, 255, .38)",
  "--step-line-background": "#fff",
  "--switch-handle-background": "#727272",
  "--switch-track-background": "#727272",
  "--tab-inactive-color": "rgba(255, 255, 255, .65)",
  "--table-background": "#303030",
  "--table-footer-border-top": "thin solid rgba(255, 255, 255, 0.12)",
  "--table-tbody-td-text-color": "#fff",
  "--table-tbody-tr-border-bottom": "thin solid rgba(255, 255, 255, 0.12)",
  "--table-tbody-tr-hover-background": "#4c4b4b",
  "--table-thead-border-bottom": "thin solid rgba(255, 255, 255, 0.12)",
  "--table-thead-th-text-color": "rgba(255, 255, 255, 0.6)",
  "--tabs-background": "#1e1e1e",
  "--time-picker-body-background": "#303030",
  "--time-picker-clock-container-background": "#545454",
  "--time-picker-clock-item-disable-color": "#aaaaaa",
  "--time-picker-clock-item-text-color": "#fff",
  "--uploader-action-background": "#303030",
  "--uploader-action-icon-color": "#fff",
  "--uploader-file-cover-background": "#303030",
  "--uploader-file-name-background": "#303030",
  "--uploader-file-name-color": "#aaa",
  "--watermark-content-color": "#ffffff",
  "--year-picker-item-color": "#fff",
}
`;
